src/pywrapper.cpp

```diff
@@ -340,6 +397,8 @@ bool PyWrapper::exec(const std::string& line, bool debug, T* val)
     PyObject* r = PyRun_String(line.c_str(), Py_single_input, globDict, locDict);
     if (r == nullptr) {
         if (PyErr_Occurred()) {
+            std::cerr << "Tried to execute string '" << line << "'" << std::endl;
```

Comment:
Leftover from debugging? ;)

Reply:
The idea was to print the "offending command" when PyERR_PRint is used (e.g. https://github.com/hz-b/PyDevice/blob/73f8ad9e9389beefbb1f90cdc8e7b7cfe0b3dcab/src/pywrapper.cpp#L393)? 

I thought that PyErr_Print would emit its messages to stdout or stderr anway. When I started to try to understand why
python3 did not work, the error messages of PyErr_Print were not really understandable for me.

Here I'd follow your preference.